trojsten/submit/views.py

```diff
@@ -499,12 +505,93 @@ def task_submit_post(request, task_id, submit_type):
             return redirect(request.POST["redirect_to"])
         else:
             return redirect(reverse("task_submit_page", kwargs={"task_id": int(task_id)}))
-
     else:
-        # Only Description and Source and Zip submitting is developed currently
         raise Http404
 
 
+@login_required
+def task_submit_post_text(request, task_id):
```

Comment:
tejto metode by sa zisli poriadne unit testy, obzvlast preto, ze je v nej dost vela if vetiev a teda sa bude zlozito testovat manualne a mala by byt pretestovana vzdy ked sa budu robit nejake zmeny v submite..

Reply:
S tymto si nevieme dat nejak rady, chcel som sa pozriet na to, ako sa testuje ta metoda task_submit_post, ale nikde to neviem najst. Neviem, ako mame simulovat odovzdanie toho formulara.

Reply:
Poprípade, mohli by sme to teraz mergnut bez toho testu, nech mozeme vypustit susi do sveta, a testy by sme dorobili neskor?